rosetta/views.py

```diff
@@ -155,6 +155,8 @@ def _request_request(key, default=None):
                         storage.set('rosetta_last_save_error', True)
 
             if file_change and rosetta_i18n_write:
+                pre_save.send(sender=None, language_code=rosetta_i18n_lang_code, request=request)
```

Comment:
I wonder if the signal shouldn't be sent before this test? I.e. what is the semantics of the `pre_save` signal? Should it only be fired if we can actually save the PO file?